source/connect-to-govwifi.html.erb

```diff
@@ -4,7 +4,28 @@ title: Connect to GovWifi
 
 <div class="container">
   <div class="grid-row">
-    <h1 class="heading-large">Connect staff to GovWifi</h1>
+    <div class="column-one-third">
+      <div>
```

Comment:
Is this extra div required?
We were also thinking of adding "<div class="column-one-third">" to the partial itself so there is less repitition

Reply:
Actually no scrap that